src/rdsn/src/runtime/security/security.thrift

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-include "../../dsn.thrift"
+include "../../../../../idl/dsn.thrift"
```

Comment:
dsn.thrift in rdsn/src/ looks fine to me, why did you move it to an outer directory?

Reply:
dsn.thrift contains some common thrift structures which will be used for not only rdsn, but also client libs. Its name 'dsn.thrift' is not accurate, maybe 'base.thrift' would be better, like it named in client lib directories(e.g. https://github.com/apache/incubator-pegasus/blob/master/java-client/idl/base.thrift).

Reply:
Of course, we will refactor the code and only leave one unique base.thrift in Pegasus later.